common/lib/xmodule/xmodule/modulestore/mixed.py

```diff
@@ -172,8 +172,12 @@ def _get_modulestore_for_courseid(self, course_id=None):
             if mapping is not None:
                 return mapping
             else:
+                if isinstance(course_id, LibraryLocator):
```

Comment:
LibraryLocator *is* a type of `CourseKey` at the moment (though I'd like that to change in the long term), so technically a course_key does not imply not a library key, here or elsewhere.

(If this method was called "_get_modulestore_for_course" it would be wrong because a course is not a library - but a course ID can be a library ID)

However, I don't like relying on technicalities and would prefer to rename this method and its argument for clarity. In split I used "courselike" to refer to "course or library". But see my other comment below just now - I think we need to define a standard term and use it consistently.

Please let me know your thoughts and what specifically you'd like changed before we can merge this PR.